inference-engine/tests/functional/plugin/myriad/ngraph/operations/static_shape_broadcast.cpp

```diff
@@ -73,6 +74,25 @@ class StaticShapeBroadcastExplicitTests
     std::shared_ptr<ngraph::opset3::Constant> m_axesMapping;
 };
 
+class StaticShapeBroadcastBidirectionalTests
```

Comment:
It's the same as `StaticShapeBroadcastNumpyTests` so please just reuse it.
`using StaticShapeBroadcastBidirectionalTests = StaticShapeBroadcastNumpyTests;`